Source/RapyutaSimulationPlugins/Public/Core/RRMeshActor.h

```diff
@@ -36,42 +36,25 @@ class RAPYUTASIMULATIONPLUGINS_API ARRMeshActor : public ARRBaseActor
 
 public:
     /**
-     * @brief ACTOR INTIALIZING GENERAL INFO (Unique name, mesh list, material list, etc.)
-     * 
-     * @tparam TActorSpawnInfo 
-     * @param InActorInfo 
-     * @return true 
-     * @return false 
-     */
-    template<typename TActorSpawnInfo>
-    bool InitializeWithSpawnInfo(const TActorSpawnInfo& InActorInfo)
-    {
-        ActorInfo = MakeShared<TActorSpawnInfo>(InActorInfo);
-
-        // ACTOR INTIALIZING GENERAL INFO (Unique name, mesh list, material list, etc.)
-        return Initialize();
-    }
-
-    /**
-     * @brief Initialize MeshActor by 1)Create child mesh components, 2)#SetCustomDepthEnabled, 3)Set mobility. 
-     * 
-     * @return true 
-     * @return false 
+     * @brief Initialize MeshActor by 1)Create child mesh components, 2)#SetCustomDepthEnabled, 3)Set mobility.
+     *
+     * @return true
+     * @return false
```

Comment:
`InitializeWithSpawnInfo()` moved to `ARRBaseActor` to be invokable also by non-template version of `SpawnSimActor()`